docs/Documenting.md

```diff
@@ -48,53 +61,59 @@ s32 foo;
 ```
 
 ## Documenting Files
-File documentation should go near the top of the file, below includes. It should only feature information that is general to the file.
+
+File documentation should be located at the top of the file, above `#include`s.
+
+File documentation should only feature information that is general to the file or the system it implements.
+
 ```c
 /**
- * @file file_name.c
+ * @file z_fcurve_data_skelanime.c
+ * @brief Curve skeleton animation system
  *
- * My description of this file
+ * A curve skeleton has a fixed number of limbs, ...
+...
  */
 ```
 
 ## Other
 
 ### Documenting Bugs:
+
 Bugs should be documented on the line above where the bug begins.
+
 ```c
-//! @bug description
+//! @bug Missing early return
 ```
+
+Bug described on multiple lines should still use the `//!` syntax, over multiple lines. For example:
+
+```c
+//! @bug this code was clearly meaning to print `abs(camera->camDataIdx)` as a
+//! one-or-two-digit number, instead of `i`.
+```
+
 ### Linking related information:
+
 `@see` should be used to provide links to related information where appropriate, for example:
+
 ```c
 /**
- * Save File Data
- * @see SaveContext
+ * Sets the next framebuffer to the framebuffer associated to `task`.
+ * If there is no current buffer or it is time to swap, this buffer will be swapped to
+ * immediately, otherwise it will be swapped to later in Sched_HandleRetrace.
+ *
+ * @see Sched_HandleRetrace
```

Comment:
Does Doxygen generate the correct links if function names are not followed by `()`?